src/components/Avatar/Avatar.tsx

```diff
@@ -70,7 +70,7 @@ export type AvatarProps = ImageAvatarProps | InitialsAvatarProps;
 /**
  * An `Avatar` shows a thumbnail representation of both an individual or group.
  */
-export default class Avatar extends React.PureComponent<AvatarProps, {}> {
+export default class Avatar extends React.Component<AvatarProps, {}> {
```

Comment:
can we leave leaf node components like Avatar to use PureComponent?

Reply:
I think it's simpler advice to give to people working in this repo to just use `Component`. Our `shouldComponentUpdate` boundary will probably be at the components connected to the store. We can measure the difference, but I think PureComponent won't give us anything at these leaf nodes.

Reply:
sorry if you're waiting for my reply, still a LGTM from my end

Reply:
Ah right, I forgot about this one.